docs/content/en/docs/getting-started/developer/_index.md

```diff
@@ -0,0 +1,79 @@
+---
+title: "K8ssandra developer quick start"
+linkTitle: "Developers"
+weight: 1
+description: |
+  Kick the tires and take K8ssandra for a spin!
+---
+
+* [Accessing K8ssandra using CQLSH]({{< relref "#access-k8ssandra-using-cqlsh" >}}): Accessing K8ssandra using the standard Cassandra CQLSH utility.
+* [Accessing K8ssandra via Stargate]({{< relref "#access-k8ssandra-using-the-stargate-api" >}}): Accessing K8ssandra using the Stargate API and the GraphQL Playground.
+
+## Access Cassandra using CQLSH
```

Comment:
This section should be reworked to utilize a local `cqlsh` install and `kubectl port-forward` to the Stargate service instead of `kubectl exec`